src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -32,6 +33,8 @@ public class ParameterProvider {
   public static final int IO_TIME_CPU_RATIO_DEFAULT = 2;
   public static final int BLOB_UPLOAD_MAX_RETRY_COUNT_DEFAULT = 24;
   public static final long MAX_MEMORY_LIMIT_IN_BYTES_DEFAULT = -1L;
+  /* parameter that enables main memory optimization for Parquet format (BdecVersion.THREE) */
+  public static final boolean ENABLE_PARQUET_MEM_OPT_DEFAULT = true;
```

Comment:
I think we should wait for the testing result before making the decision on the default

Reply:
I agree. I think we can merge with the parameter set to `false`.